config/default.yml

```diff
@@ -4108,6 +4108,12 @@ Style/LambdaCall:
     - call
     - braces
 
+Style/LengthyLiteral:
+  Description: 'Checks for extremely large literals, such as `Array`s and `Hash`es with many entries.'
+  Enabled: pending
+  VersionAdded: '<<next>>'
+  LengthThreshold: 250
```

Comment:
I picked this arbitrarily. 🤷 

We shouldn't pick a number so high that it makes the rule useless, but also shouldn't be too aggressive in our default. Application owners are free to adjust the number as they see fit.